docs/images/qr.svg

```diff
@@ -0,0 +1,686 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
If you have time, could you run this code through [SVGOMG](https://jakearchibald.github.io/svgomg/)? This simplifies file a lot.

Reply:
There's also Google Charts APIs: https://chart.googleapis.com/chart?chs=300x300&cht=qr&chl=https://github.com/prinz-nussknacker/prinz

Reply:
looks better, will change